docs/examples/ex_slepc.py

```diff
@@ -0,0 +1,38 @@
+from sys import argv
+
+import numpy as np
+
+import slepc4py
+from petsc4py import PETSc
+from slepc4py import SLEPc
+
+import ex16
+
+
+def petsc_mat(A):
+    return PETSc.Mat().createAIJ(
+        size=A.shape,
+        csr=(A.indptr, A.indices, A.data))
```

Comment:
Add this as `COOData.to_petsc`?

Reply:
Sure.  I'm not expert in PETSc and haven't touched this since, but it did work.  Should it go in #474 or #596?